msg/json/en.json

```diff
@@ -39,6 +39,7 @@
     "EVENT_WHENFLAGCLICKED": "when %1 clicked",
     "EVENT_WHENTHISSPRITECLICKED": "when this sprite clicked",
     "EVENT_WHENSTAGECLICKED": "when stage clicked",
+    "EVENT_WHENTOUCHINGOBJECT": "when this sprite touches %1",
```

Comment:
Same question here as for `msg/js/en.js`

Reply:
We agreed to leave the changes in scratch-blocks for the new event block as Eric is still hoping to get it in before the freeze. This way it'll already be in the translations.

Reply:
Ah, I wasn't actually referring to that particular line, just the same question about the en.json file as with the en.js file, which you've already answered.